web/src/lib/domain-logic/businessPersonaHelpers.test.ts

```diff
@@ -0,0 +1,216 @@
+import {
+  isNexusBusiness,
+  isOwningBusiness,
+  isRemoteWorkerOrSellerBusiness,
+  isStartingBusiness,
+} from "@/lib/domain-logic/businessPersonaHelpers";
+import {
+  Business,
+  ForeignBusinessTypeId,
+  generateBusiness,
+  generateProfileData,
+} from "@businessnjgovnavigator/shared/";
+
+describe("businessPersonaHelpers", () => {
+  describe("isOwningBusiness", () => {
+    it("returns false when business is undefined", () => {
+      expect(isOwningBusiness(undefined)).toBe(false);
+    });
+
+    it("returns false when business persona is starting", () => {
+      expect(
+        isOwningBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "STARTING",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+
+    it("returns false when business persona is foreign", () => {
+      expect(
+        isOwningBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "FOREIGN",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+
+    it("returns true when business persona is owning", () => {
+      expect(
+        isOwningBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "OWNING",
+            }),
+          })
+        )
+      ).toBe(true);
+    });
+  });
+
+  describe("isStartingBusiness", () => {
+    it("returns false when business is undefined", () => {
+      expect(isStartingBusiness(undefined)).toBe(false);
+    });
+
+    it("returns false when business persona is owning", () => {
+      expect(
+        isStartingBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "OWNING",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+
+    it("returns false when business persona is foreign", () => {
+      expect(
+        isStartingBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "OWNING",
+            }),
+          })
+        )
+      ).toBe(false);
+    });
+
+    it("returns true when business persona is starting", () => {
+      expect(
+        isStartingBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              businessPersona: "STARTING",
+            }),
+          })
+        )
+      ).toBe(true);
+    });
+  });
+
+  describe("isNexusBusiness", () => {
+    describe("foreign business persona", () => {
```

Comment:
[sand] Why the nested describe block?

Reply:
I removed them, I had the describe block because of how the data was staged. I agree it's unnecessary.